python/langsmith/run_helpers.py

```diff
@@ -39,19 +39,24 @@
     from langchain.schema.runnable import Runnable
 
 logger = logging.getLogger(__name__)
-_PARENT_RUN_TREE = contextvars.ContextVar[Optional[run_trees.RunTree]](
+_PARENT_RUN_TREE = contextvars.ContextVar[Optional[run_trees.Span]](
     "_PARENT_RUN_TREE", default=None
 )
 _PROJECT_NAME = contextvars.ContextVar[Optional[str]]("_PROJECT_NAME", default=None)
 _TAGS = contextvars.ContextVar[Optional[List[str]]]("_TAGS", default=None)
 _METADATA = contextvars.ContextVar[Optional[Dict[str, Any]]]("_METADATA", default=None)
 
 
-def get_current_run_tree() -> Optional[run_trees.RunTree]:
+def get_current_span() -> Optional[run_trees.Span]:
     """Get the current run tree."""
     return _PARENT_RUN_TREE.get()
 
 
+def get_parent_span() -> Optional[run_trees.Span]:
+    """Get the parent run tree."""
+    return _PARENT_RUN_TREE.get()
```

Comment:
why do these two methods do the same thing?

Reply:
copy pasta